app/views/users/index.html.erb

```diff
@@ -20,7 +20,7 @@
         <td><%= user.full_name || "--" %></td>
         <td><%= user.id_string %></td>
         <td><%= @course.role_string_for_user(user) %></td>
-        <% if @course.role_string_for_user(user) != "Student" %>
+        <% if ((@course.role_string_for_user(user) != "Student") && (@course.role_string_for_user(user) != "Teaching Staff")) %>
```

Comment:
It's good to include this check. Just that our codebase had pre-defined constants for roles, please use it.